ci-operator/step-registry/ipi/install/heterogeneous/day2-kerneltype/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
+  - sergiordlr
+  - rioliu-rh 
+  - aleskandro
```

Comment:
while you are on this... you can remove my nick from the OWNERS including multiarch-approvers/reviewers. I'm already on them.

Reply:
Done